docs/en/Storage.md

```diff
@@ -1,132 +1,261 @@
 ```
 
-Now open the
-[MinIO browser](https://minimal-tenant1-minio.covid.cloud.statcan.ca) and you
-will see your test file there!
+Now open the [MinIO Web Portal](#minio-web-portal) or browse to
+`~/minio/minimal-tenant1/private` to see your test file!
+
+<!-- prettier-ignore -->
+??? tip "`mc` can do a lot" 
+    In addition to copying files, `mc` can do a lot more (like `mc ls minio-minimal/FIRSTNAME-LASTNAME` to list the contents of a bucket).  Check out the [mc docs](https://docs.min.io/docs/minio-client-complete-guide.html) or run `mc --help` for more information.
+
+<!-- prettier-ignore -->
+??? tip "See the example notebooks!"
+    There is a template provided for connecting in `R`, `python`, or by the
+    command line, provided in [jupyter-notebooks/self-serve-storage](https://github.com/StatCan/jupyter-notebooks/tree/master/self-serve-storage) (also auto-mounted to all jupyter notebook servers in `~/jupyter-notebook`). You can
+    copy-paste and edit these examples! They should suit most of your needs.
+
+#### Other S3-Compliant Methods
+
+MinIO is S3 compliant - it uses the same standard as Amazon S3 and other bucket
+services. Tools designed to use S3 will generally also work with MinIO, for
+example Python packages and instructions on how to access files from S3. Some
+examples of this are shown in
+[jupyter-notebooks/self-serve-storage](https://github.com/StatCan/jupyter-notebooks/tree/master/self-serve-storage).
+
+### Bucket Types and Access Scopes
+
+Two types of buckets are available:
+
+- **[Minimal](https://minimal-tenant1-minio.covid.cloud.statcan.ca):**  
+  By default, use this one. It is backed by an SSD and provides a good balance
+  of cost and performance.
+- **[Premium](https://premium-tenant1-minio.covid.cloud.statcan.ca):**  
+  Use this if you need high read/write speeds and don't mind paying ~2x the
+  storage cost. These are somewhat faster than the minimal storage.
+
+Generally if you aren't sure which you need, start with **Minimal**. You can
+always change your mind if you see your work limited by file transfer speeds.
+
+Within each bucket type, everyone has two storage locations they can use, each
+providing different access scopes:
+
+- Private:
+  - Accessible only by someone within your namespace (typically only by you from
+    your own notebook servers/remote desktop, unless you're working in a
+    [shared namespace](./Collaboration.md#requesting-a-namespace))
+  - Examples:
+    - mounted as a drive in notebook server:
+      `~/minio/minimal-tenant1/private/myfolder/myfile`
+    - via `mc` tool/MinIO portal: `firstname-lastname/myfolder/myfile`
+- Shared
+  - Writable only by you, but readable by anyone with access to the platform.
+    Great for sharing public data across teams
+  - Examples:
+    - mounted as a drive in notebook server:
+      `~/minio/minimal-tenant1/shared/myfolder/myfile`
+    - via `mc` tool/MinIO portal: `shared/firstname-lastname/myfolder/myfile`
```

Comment:
Silly thing, but I think this will look better as a 2-column table. 

| Private | Shared | 
|:----------:|:----------:|
| Accessible only by someone within your namespace (typically only by you from your own notebook servers/remote desktop, unless you're working in a [shared namespace](./Collaboration.md#requesting-a-namespace)) | Writable only by you, but readable by anyone with access to the platform. Great for sharing public data across teams |
| Examples | 
| mounted as a drive in notebook server:  `~/minio/minimal-tenant1/private/myfolder/myfile` | mounted as a drive in notebook server:  `~/minio/minimal-tenant1/shared/myfolder/myfile` |
|  via `mc` tool/MinIO portal: `firstname-lastname/myfolder/myfile` |  via `mc` tool/MinIO portal: `shared/firstname-lastname/myfolder/myfile` |